source/tutorials/profiling-gpu/profiling-gpu.rst

```diff
@@ -0,0 +1,134 @@
+Profiling GPU
+########################################################################
+
+HW pre-requisites
+===========================
+Build machine/PC with Ubuntu 16.04
+
+USB drive 8GB, ext4 formatted.
+
+USB type A connection from build machine to target.
```

Comment:
Add comment that the cable is used only adb is enabled on the specific USB port.